query-service/build.gradle.kts

```diff
@@ -1,8 +1,15 @@
 plugins {
   java
   application
+  jacoco
   id("org.hypertrace.docker-java-application-plugin") version "0.8.1"
   id("org.hypertrace.docker-publish-plugin") version "0.8.1"
+  id("org.hypertrace.integration-test-plugin")
+  id("org.hypertrace.jacoco-report-plugin")
+}
+
+repositories {
+  maven("http://packages.confluent.io/maven")
```

Comment:
Should we think about adding this to https://github.com/hypertrace/hypertrace-gradle-repository-plugin?